source/includes/_tipser-elements.md

```diff
@@ -870,3 +870,30 @@ addToCartAndGoToCheckout(productId).then(() => {
 ```
 
 Adds to cart a product with a given Tipser product id and then opens the checkout modal dialog. Alternatively, redirects to the URL defined in `customUrls.checkoutUrl` configuration option if specified. Returns a promise that will succeed or reject depending on the status of that operation. 
+
+
+
+## Advanced usages of Elements
```

Comment:
Generalnie to mamy już sekcję advanced usage: 
![image](https://user-images.githubusercontent.com/1180737/111992402-ac867880-8b15-11eb-8d3d-24af1dc1929a.png)

Nie chciałaś tego umieścić w tamtej sekcji?